src/02 - Object types.md

```diff
@@ -454,6 +455,26 @@ The following notation is used in the tables below:
 |      | ref    | refRegistrering        |    x     |             |           |  |      |  x   |   x   |      |       x       | <p>&bull; references **Basisregistrering** or **Journalpost**</p><p>&bull; may be partially supported by systems</p><p>&bull; *unlink* not supported in Documaster</p> | &nbsp;    |
 
+### Dokumentflyt
+
+| ID   | Type   | Field                  | Not null | Has default | Code list |  | Save | View | Query | Sort | Link / Unlink | Comment 1                                          | Comment 2 |
+|:-----|:-------|:-----------------------|:--------:|:-----------:|:---------:|:-|:----:|:----:|:-----:|:----:|:-------------:|:---------------------------------------------------|:----------|
+|      | string | id                     |    x     |      x      |           |  |  sm  |  x   |   x   |  x   |               |                                                    |           |
+|      | string | version                |    x     |      x      |           |  |  sm  |  x   |   x   |      |               |                                                    |           |
+|      |        |                        |          |             |           |  |      |      |       |      |               |                                                    |           |
+| M001 | string | uuid                   |    x     |      x      |           |  |  cw  |  x   |   x   |      |               |                                                    |           |
+| M600 | date   | opprettetDato          |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                    |           |
+| M601 | string | opprettetAv            |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                    |           |
+| M601 | string | opprettetAvBrukerIdent |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                    |           |
+| M660 | string | flytTil                |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M660 | string | flytTilBrukerIdent     |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M661 | date   | flytMottattDato        |    x     |             |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
+| M662 | date   | flytSendtDato          |    x     |      x      |           |  |  x   |  x   |   x   |  x   |               |                                                    |           |
```

Comment:
Field `flytSendtDato` does not have a default value.

Reply:
Fixed. We had done internal model changes and I had not updated it.